src/client/public/css/pages/sign-up.css

```
@@ -3,3 +3,61 @@
 	font-size: var(--font-size-200);
 	margin: 5px 0 3px 5px;
 }
+
+.wrapper {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	height: 100vh;
```

```suggestion
	flex: 1
```

body is 100vh already, I'll write to change it to Sofia too